AdobeReader/AdobeReader.munki.recipe

```diff
@@ -1,143 +1,144 @@
-                <dict>
-                    <key>warning_message</key>
-                    <string>This recipe will soon be removed. Please remove it from your list of recipes.</string>
-                </dict>
-            </dict>
-            <dict>
-                <key>Arguments</key>
-                <dict>
-                    <key>flat_pkg_path</key>
-                    <string>%pathname%/*.pkg</string>
-                    <key>destination_path</key>
-                    <string>%RECIPE_CACHE_DIR%/pkg_unpack</string>
-                    <key>purge_destination</key>
-                    <true/>
-                </dict>
-                <key>Processor</key>
-                <string>FlatPkgUnpacker</string>
-            </dict>
-            <dict>
-                <key>Arguments</key>
-                <dict>
-                    <key>pkgroot</key>
-                    <string>%RECIPE_CACHE_DIR%/application_payload/Applications</string>
-                    <key>pkgdirs</key>
-                    <dict/>
-                </dict>
-                <key>Processor</key>
-                <string>PkgRootCreator</string>
-            </dict>
-            <dict>
-                <key>Arguments</key>
-                <dict>
-                    <key>pkg_payload_path</key>
-                    <string>%RECIPE_CACHE_DIR%/pkg_unpack/application.pkg/Payload</string>
-                    <key>destination_path</key>
-                    <string>%RECIPE_CACHE_DIR%/application_payload/Applications</string>
-                    <key>purge_destination</key>
-                    <true/>
-                </dict>
-                <key>Processor</key>
-                <string>PkgPayloadUnpacker</string>
-            </dict>
-            <dict>
-                <key>Arguments</key>
-                <dict>
-                    <key>faux_root</key>
-                    <string>%RECIPE_CACHE_DIR%/application_payload</string>
-                    <key>installs_item_paths</key>
-                    <array>
-                        <string>/Applications/Adobe Reader.app</string>
-                    </array>
-                </dict>
-                <key>Processor</key>
-                <string>MunkiInstallsItemsCreator</string>
-            </dict>
-            <dict>
-                <key>Processor</key>
-                <string>MunkiPkginfoMerger</string>
-            </dict>
-            <dict>
-                <key>Arguments</key>
-                <dict>
-                    <key>pkg_path</key>
-                    <string>%pathname%</string>
-                    <key>repo_subdirectory</key>
-                    <string>%MUNKI_REPO_SUBDIR%</string>
-                </dict>
-                <key>Processor</key>
-                <string>MunkiImporter</string>
-            </dict>
-        </array>
-    </dict>
+</string>
+			<key>unattended_install</key>
+			<true/>
+		</dict>
+	</dict>
+	<key>MinimumVersion</key>
+	<string>0.2.0</string>
```

Comment:
```suggestion
	<string>1.4</string>
```

Reply:
I'm not against updating the MinimumVersion on these, but I'm curious if it's _needed_, and if it is, how you came to version 1.4.

Reply:
Because it's the minimum version for the parent recipe, so theoretically all child recipes would at least require the same minimum version. It's possible that the download recipe has an incorrect assumption on minimum version in which case it could be updated, at which point the others could be adjusted to use it as the baseline.

Probably not an essential/blocking issue, just something I noticed while looking at the changes.

Reply:
Interesting observation, thanks for pointing it out. I'm good with bumping these all to 1.4.